src/views/privacyPolicy/index.tsx

```diff
@@ -0,0 +1,419 @@
+import { Box, Typography } from "@mui/material"
+import { styled } from "@mui/system"
+
+const Container = styled(Box)(({ theme }) => ({
+  padding: "12rem 0",
+  [theme.breakpoints.down("md")]: {
+    padding: "8rem 1.6rem",
+  },
+  "& *": {
+    fontSize: "2rem",
+    textAlign: "justify",
+    [theme.breakpoints.down("md")]: {
+      fontSize: "1.6rem",
+    },
+  },
+  "& h1, & h2, & h3": {
+    fontWeight: "bold",
+    marginBottom: "2.8rem",
+    [theme.breakpoints.down("md")]: {
+      marginBottom: "2rem",
+    },
+  },
+  "& p": {
+    marginBottom: "1.6rem",
+    "+ h1, + h2, + h3": {
+      marginTop: "2.8rem",
+      [theme.breakpoints.down("md")]: {
+        marginTop: "2rem",
+      },
+    },
+  },
+  "& ul": {
+    listStyle: "inside",
+    "& li": {
+      margin: "1.6rem 0",
+      [theme.breakpoints.down("md")]: {
+        margin: "1rem 0",
+      },
+    },
+    [`& ${TitleTypography}`]: {
+      textAlign: "center",
+    },
+    "+ h1, + h2, + h3": {
+      marginTop: "2.8rem",
+      [theme.breakpoints.down("md")]: {
+        marginTop: "2rem",
+      },
+    },
+  },
+}))
+
+const TitleTypography = styled(Typography)(({ theme }) => ({
+  textAlign: "center",
+  margin: "2rem auto",
+  lineHeight: 1.5,
+}))
+
+const DashTypography = styled("h3")(({ theme }) => ({
+  "&::before": {
+    content: '"-"',
+    margin: "0 3rem",
+    fontWeight: "normal",
+    [theme.breakpoints.down("md")]: {
+      margin: "0 2rem",
+    },
+  },
+}))
+
+const PrivacyPolicy = () => {
+  return (
+    <Container className="wrapper">
+      <TitleTypography variant="h1">Privacy Policy For Scroll Limited</TitleTypography>
```

Comment:
@zzq0826 can we make typography a bit more consistent, e.g. https://scroll.io/privacy-policy vs https://scroll.io/terms-and-conditions ?